cfgmgr/buffermgrdyn.h

```diff
@@ -0,0 +1,245 @@
+#ifndef __BUFFMGRDYN__
+#define __BUFFMGRDYN__
+
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "orch.h"
+
+#include <map>
+#include <set>
+#include <string>
+
+namespace swss {
+
+#define INGRESS_LOSSLESS_PG_POOL_NAME "ingress_lossless_pool"
+#define DEFAULT_MTU_STR             "9100"
```

Comment:
Currently, SAI default port MTU is 1514. We could maintain the same default MTU instead of jumbo mtu size.

Reply:
But the SONiC MTU is 9100. So we need to align it with SONiC